### PR TITLE
Added options to disable old TLS versions

### PR DESCRIPTION
**Pull request by nvidianz**

### CODE REVIEW

File: nvflare/tool/package_checker/utils.py [hunk 1]
```diff
@@ -171,6 +171,7 @@ def check_socket_server_running(startup: str, host: str, port: int) -> bool:
     try:
         # SSL communication
         ctx = ssl.create_default_context()
+        ctx.minimum_version = ssl.TLSVersion.TLSv1_2
```
[the remark this answers is not in the document]
Comment (nvidianz) [reply nvflare/tool/package_checker/utils.py:174]: Unit-test requires certs. It can't be done easily. Will add test later.

**Comment by YuanTingHsieh** [comment]
@nvidianz you need to either update the branch or rebase it on latest dev thanks.

**Pull request merged**